### ISSUE TITLE
Need NodeBB 0.6.x compatibility

### ISSUE THREAD

**Issue by tedr56** [arasbm/nodebb-plugin-whoisin#1]
I like this plugin.

Too bad I can't install it on new NodeBB version :)

**Comment by arasbm** [comment]
yes you can! the latest version works with nodebb 0.6.0 or higher! You may
get a warning, but should be able to install it. Let me know if there a any
issues.
